ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/LocalStateQuery/Server.hs

```diff
@@ -50,7 +50,7 @@ import           Ouroboros.Network.Protocol.LocalStateQuery.Examples
                      (localStateQueryClient)
```

Comment:
This PR currently has some unrelated "integration" changes, since it depends on an upstream commit of `ouroboros-network`.

Once `main` is updated, this PR can be collapsed.

Reply:
This has already been updated on `main`, you can rebase on top of it to get eg this change (the PR branch is currently 61 commits behind `main`).

Reply:
rebased --- I should pull that remaining commit into its own PR